### PR TITLE
Add microservice database config

### PR DESCRIPTION
**Pull request by Mykhailenko**

### CODE REVIEW

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 1]
```diff
@@ -91,8 +94,31 @@ All configuration files of ProActive Scheduler can be found under `PROACTIVE_HOM
...
+|Scheduling-api microservice|dist/war/scheduling-api/WEB-INF/classes/application.properties|<<Database configuration>>
+|Job-planner microservice|dist/war/job-planner/WEB-INF/classes/application.properties|<<Database configuration>>
+|Catalog microservice|dist/war/catalog/WEB-INF/classes/application.properties|<<Database configuration>>
+
 |===
 
+=== Database configuration
+
+Scheduler, Resource Mananger, and three microservices require to have direct access to database.
+Thus they have to be properly configured to have access to database.
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: Thus, they must each have a correct database configuration.

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -91,8 +94,31 @@ All configuration files of ProActive Scheduler can be found under `PROACTIVE_HOM
...
+
+To configure Scheduler or Resource Mananger, one have to modify config/scheduler/database.properties
+and config/rm/database.properties respectively. It is necaessary to configure five following preperties:
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: The following five properties must be configured:

Comment (Mykhailenko) [reply src/docs/admin/ProActiveAdminGuide.adoc]: thanks

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 1 above)
Comment (Mykhailenko) [reply src/docs/admin/ProActiveAdminGuide.adoc]: Thanks

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -91,8 +94,31 @@ All configuration files of ProActive Scheduler can be found under `PROACTIVE_HOM
...
+`hibernate.connection.driver_class`, `hibernate.connection.url`, `hibernate.dialect`,
+`hibernate.connection.username`, and `hibernate.connection.password`.
+
+
+Microservice database configuration is always in `/dist/war/MICROSERVICE-NAME/classes/applicaiton.properties`.
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: For each microservice, a microservice configuration file can be found in: 
`/dist/war/MICROSERVICE-NAME/classes/application.properties`

The job-planner, catalog and scheduling-api microservices must contain a database configuration. In their application.properties file, you need to set the following five properties:

Comment (Mykhailenko) [reply src/docs/admin/ProActiveAdminGuide.adoc]: Thanks

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 4]
```diff
@@ -91,8 +94,31 @@ All configuration files of ProActive Scheduler can be found under `PROACTIVE_HOM
...
+
+Scheduler, Resource Mananger, and three microservices require to have direct access to database.
+Thus, they must each have a correct database configuration.
+
+To configure Scheduler or Resource Mananger, one have to modify `config/scheduler/database.properties`
+and `config/rm/database.properties` respectively. The following five properties must be configured:
+`hibernate.connection.driver_class`, `hibernate.connection.url`, `hibernate.dialect`,
+`hibernate.connection.username`, and `hibernate.connection.password`.
+
+For each microservice, a microservice configuration file can be found in: `/dist/war/MICROSERVICE-NAME/classes/applicaiton.properties`.
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: application.properties

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 5]
```diff
@@ -91,8 +94,31 @@ All configuration files of ProActive Scheduler can be found under `PROACTIVE_HOM
...
+|Resource Manager configuration|config/rm/database.properties|<<Database configuration>>
+|Scheduling-api microservice|dist/war/scheduling-api/WEB-INF/classes/application.properties|<<Database configuration>>
+|Job-planner microservice|dist/war/job-planner/WEB-INF/classes/application.properties|<<Database configuration>>
+|Catalog microservice|dist/war/catalog/WEB-INF/classes/application.properties|<<Database configuration>>
+
 |===
 
+=== Database configuration
+
+Scheduler, Resource Mananger, and three microservices require to have direct access to database.
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: to the database.

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 4 above)
Comment (Mykhailenko) [reply src/docs/admin/ProActiveAdminGuide.adoc]: ohh thanks

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 5 above)
Comment (Mykhailenko) [reply src/docs/admin/ProActiveAdminGuide.adoc]: done

**Pull request merged**